extensions/common/http/jetty-core/src/main/java/org/eclipse/edc/web/jetty/JettyExtension.java

```diff
@@ -31,9 +32,10 @@
 import java.security.NoSuchAlgorithmException;
 import java.security.cert.CertificateException;
 
-@Provides({ WebServer.class, JettyService.class })
+@Provides({WebServer.class, JettyService.class})
```

Comment:
The default formatting for static array definitions should have the whitespace after open bracket and before closing bracket